lib/phoenix_ecto.ex

```diff
@@ -5,4 +5,12 @@ defmodule Phoenix.Ecto do
   It implements many protocols that makes it easier to use
   Ecto with Phoenix either when working with HTML or JSON.
   """
+  use Application
+  import Supervisor.Spec
+
+  def start(_type, _args) do
+    Supervisor.start_link([
+      supervisor(Phoenix.Ecto.SQL.SandboxSupervisor, [])
+    ], strategy: :one_for_one, name: DockYard.Supervisor)
```

Comment:
:P

Reply:
I think you want this to be a simple one for one, otherwise start_child will fail.

Reply:
![marketing](https://i.imgflip.com/1w2cg1.jpg)